### ISSUE TITLE
New version available message doesn't honor "Don't show again" 

### ISSUE THREAD

**Issue by markg85** [sqlitebrowser/sqlitebrowser#313]
Hi,

Every time i start SQLite Browser it pops up with "New version available". I can click "OK" and "Don't show again". Clicking OK obviously warrants the popup to show again when i start the application again (till i update), but the "Don't show again" button should just stop me from seeing the "New version available" popup. Also after restarts. Right now it seems to behave the same as clicking OK since restarting the application will still show me the popup...

And yes, if i update the popup is obviously gone, but that isn't the point of this bug report :)

Regards,
Mark

**Comment by schdub** [comment]
@markg85, hello. Thank you for your post.
It's odd, because I just checked and everything works.
Please show what values you have in the registry key:
HKCU/Software/sqlitebrowser/sqlitebrowser/checkversion

**Comment by markg85** [comment]
Now that is interesting!
The values are:

disable : true
major : 3
minor : 6
patch : 6 <-- weird one! It's in the future (3.6.0 is the latest version at this moment).

I've had these same symptoms on two computers. I can only access one, but i suspect the other one also has a patch version in the future. Setting the patch value to 0 seems to fix it for me (for now).

**Comment by justinclift** [comment]
Oh, that's weird.  It sounds like there's a bug in our version setting bits on windows. :frowning:

**Comment by MKleusberg** [comment]
We've changed the automatic update check recently because it contained several bugs. The major/minor/patch keys aren't used anymore in the current version (changed in commit da6d8ba385d18a2d9f5eafa3d0f2b9153e880d49), so they shouldn't cause any trouble... I think...
The reason for the 3.6.6 value is that one of the bugs was a typo where the value of the minor part was accidentally written into the patch key, this was fixed in 2c25b7ecb52944db225516a9b41c9f6c1860e74.

So what this means is that, in theory, the bug which is causing all this trouble should be fixed. And because the registry keys were renamed, the bad values you observed shouldn't be used anymore in version 3.6.0. I'm not sure if you're already using this version, but if you do it seems like we're definitely having more bugs to fix here...
